docs/vgclog_examples.md

```diff
@@ -0,0 +1,708 @@
+
+<trace-info objectcount="1936" scancount="1698" scanbytes="61200" />
+
+<finalization candidates="389" enqueued="1" />
+
+<ownableSynchronizers candidates="5076" cleared="523" />
+
+<references type="soft" candidates="18420" cleared="0" enqueued="0" dynamicThreshold="32" maxThreshold="32" />
+
+<references type="weak" candidates="19920" cleared="114" enqueued="60" />
+
+<references type="phantom" candidates="671" cleared="50" enqueued="50" />
+
+<stringconstants candidates="40956" cleared="109" />
+
+<object-monitors candidates="182" cleared="51" />
+
+</gc-op>
+
+<gc-op id="12385" type="classunload" timems="1.607" contextid="12364" timestamp="2020-10-18T13:35:44.609">
+
+<classunload-info classloadercandidates="425" classloadersunloaded="6" classesunloaded="2" anonymousclassesunloaded="1" quiescems="0.000" setupms="1.581" scanms="0.019" postms="0.007" />
+
+</gc-op>
+
+<gc-op id="12386" type="sweep" timems="9.464" contextid="12364" timestamp="2020-10-18T13:35:44.618" />
+```
+
+The end of the increment is recorded with `<gc-end>` and provides another snapshot of memory in the heap, similar to `<gc-start>`. 
+
+
+```xml
+<gc-end id="12387" type="global" contextid="12364" durationms="24.220" usertimems="86.465" systemtimems="0.000" stalltimems="2.846" timestamp="2020-10-18T13:35:44.618" activeThreads="4">
+
+<mem-info id="12388" free="650476504" total="1073741824" percent="60">
+
+<mem type="nursery" free="235516088" total="268435456" percent="87">
+
+<mem type="allocate" free="235516088" total="241565696" percent="97" />
+
+<mem type="survivor" free="0" total="26869760" percent="0" />
+
+</mem>
+
+<mem type="tenure" free="414960416" total="805306368" percent="51" micro-fragmented="98245682" macro-fragmented="0">
+
+<mem type="soa" free="374694688" total="765040640" percent="48" />
+
+<mem type="loa" free="40265728" total="40265728" percent="100" />
+
+</mem>
+
+<pending-finalizers system="1" default="0" reference="515" classloader="0" />
+
+<remembered-set count="13554" />
+
+</mem-info>
+
+</gc-end>
+```
+
+60% of the heap now contains free memory as a result of the final global cycle increment, which is split between the following areas of the heap:
+
+- The nursery area, which gained 0.9 MB of free memory. The nursery area now has 224.6 MB (235,516,088B) available as free memory. At the start of the final increment, the nursery area had 223.7 MB (234,609,440B) of free memory available.
+- The tenure area, which gained 355.2 MB (372,521,216B) of free memory. (the tenure area now has 395.7 MB (414,960,416B) available as free memory. At the start of the final increment, the tenure area had 40.5 MB (42,439,200B) of free memory available).
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** The global GC cycle runs to reclaim memory in the tenure area. The freeing up of memory in the nursery area is achieved by using the partial GC cycle. For more information, see [`gencon` Policy(default)](gc.md#gencon-policy-(default)).
```

Comment:
Please fix link: gc.md#gencon-policy-default and title **`gencon` policy (default)**